lib/Mojolicious/resources/public/mojo/highlight.js/highlight-mojo-dark.css

```diff
@@ -0,0 +1,107 @@
+﻿/*
+
+Based on Dark style by Ivan Sagalaev
+and prettify-mojo-dark.css by Sebastian Reidel
+
+*/
```

Comment:
That comment seems redundant.

Reply:
This was from the old PR, we can drop this too.

Reply:
The [base theme's history](https://github.com/highlightjs/highlight.js/commits/master/src/styles/dark.css) suggest that the main style classes are already set, what's only changed/removed are for language-specific styles that crept into the base theme during its development.  I doubt this will change any further given the last commit for that base theme is from 2015.